### PR TITLE
7 gls feedback ws

### PR DESCRIPTION
**Pull request by bealdav**
Allow to save data about web service exchange in attachment

gls_colis

Allow to send these data to the web service provider in case of logistics incident

### CODE REVIEW

**Comment by bealdav** [comment]
@alexis-via  @guewen do you think we could have the same principle for Coliposte V8 international ?
thanks for a review

for Coliposte we need to store xml soap request. Call me for more info

**Comment by Viggor** [comment]
Works fine with french partners, but with foreign partners there is a exception in GLS lib.

File: delivery_carrier_label_gls/stock.py [hunk 1]
```diff
@@ -250,6 +259,40 @@ def _generate_gls_labels(
...
+            'file_type': 'text/plain',
+        }
+        return self.pool['ir.attachment'].create(
+            cr, uid, data, context=context)
+
+    def _record_ws_exchange(self, label, pack):
+        trac_infos = ''
+        if 'raw_response' in label and 'request' in label:
+            trac_infos = (
+                u'Séquence Colis GLS:\n====================\n%s \n\n'
```
Comment (yvaucher) [inline delivery_carrier_label_gls/stock.py:288]: Is it wanted to have Séquence in french ?

**Comment by yvaucher** [comment]
Only a little question otherwise :+1:

**Comment by bealdav** [comment]
see https://github.com/OCA/carrier-delivery/pull/91

**Pull request closed without merging**